imports/plugins/custom/sms/client/actions/settings.js

```diff
@@ -0,0 +1,50 @@
+import Alert from "sweetalert2";
+import { i18next } from "/client/api";
+
+export default {
+
+  /**
+   * Save email settings
+   * @param {Object} settings - object of mail provider settings
+   * @param {Function} callback - optional callback
+   * @return {Boolean} returns true if all fields provided and update method called
+   */
+  saveSettings(settings, callback) {
+    const { apiKey, apiToken, smsProvider, smsPhone } = settings;
+
+    if (!apiKey) {
+      Alert("Warning", "Please add an API Key", "error");
```

Comment:
Is there a good resource for making this easier, instead of going through all the language files?